tests/gui/test_02_import_data.py

```diff
@@ -52,7 +47,7 @@ def test_page_02_loads_sample_data():
     at.run()
 
     # User clicks Load Sample Data button
-    at.button("load_sample_data").click().run()
+    at.button[1].click().run()
```

Comment:
Please give a key to the button instead so the test works if the button order changes.